examples/survival_analysis/survival_analysis.ipynb

```diff
@@ -28,13 +28,15 @@
    "metadata": {},
```

Comment:
<p>This should be <code>sigma</code>  instead of <code>sd</code>  . Also note that you can use <code>dims</code>  in deterministic variables too, even if only to have nice names in the inferencedata result. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>